frontend/src/app/application-forms/validators/error-message.component.ts

```diff
@@ -11,6 +11,7 @@ export class ErrorMessageComponent {
   @Input() name: string;
 
   parseErrors(errors) {
+    console.log(errors);
```

Comment:
Do you still need this debugging log?

Reply:
@lauraGgit added task in sprint-8 story #102